web/themes/joinup/prototype/sass/_drupal-hacks.sass

```diff
@@ -182,3 +182,29 @@ form .description
     &:before
       +icon-size(15px)
       content: '\e617'
+
+// Additional styling for dropbutton.
```

Comment:
The first button does not receive classes in the same way as the second one, which makes it unnoticeable when tabbing:

![edit_buttons](https://user-images.githubusercontent.com/1923476/28863007-80724da2-7767-11e7-847e-eb825b361065.gif)